platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -93,12 +94,16 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(LandingPage.class);
 
+  @SuppressWarnings("squid:S2226" /* Lifecycle managed by blueprint. */)
   private String logo;
 
+  @SuppressWarnings("squid:S2226" /* Lifecycle managed by blueprint. */)
```

Comment:
It looks like this is a different problem with a different squid `S1068`. Same for the other two suppressions below.

Reply:
After taking out the class suppression they all have the `Squid:S2226` bug. `Squid:S1068` looks like it is just a warning and out of the scope of my ticket. Another reason I don't think I should mess with the `Squid:S1068` variable not used warnings is that they may be used in blueprint.

Reply:
What I meant was the issue with `logoToUse`, `background`, and `foreground` is not `Servlets should not have mutable instance fields`, which has a squid of `S2226`. It's "`Unused "private" fields should be removed`, which has a squid of `S1068`. I'm not saying you should address that issue just that the suppression you have isn't doing anything.